nautobot/project-static/css/base.css

```diff
@@ -791,6 +793,147 @@ td span.hover_copy:hover .hover_copy_button {
+    position: relative;
+}
+.navbar-fixed-left.collapsed ~ .main-content {
+    margin-left: 0px; 
+}
+/* When navbar is expanded, show the search form */
+.navbar-fixed-left:not(.collapsed) .navbar-search-form {
+    display: block;
+    transition-duration: var(--navbar-transition-duration);
+    transition-timing-function: ease-in-out;
+}
+/* Hide the search form when navbar collapsed */
+.navbar-fixed-left .navbar-search-form {
+    display: none;
+    transition-duration: var(--navbar-transition-duration);
+    transition-timing-function: ease-in-out;
+}
+.navbar-inverse .navbar-collapse, .navbar-inverse .navbar-form {
+    border-color: transparent;
+}
+.navbar-toggler:hover {
+    cursor: pointer;
+    background: #007dff;
+}
+.navbar-toggler {
+    -moz-appearance: none;
+    -ms-appearance: none;
+    appearance: none;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+    -webkit-box-pack: center;
+    -ms-flex-pack: center;
+    justify-content: center;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+    white-space: nowrap;
+    vertical-align: middle;
+    outline: 2px solid transparent;
+    outline-offset: 2px;
+    line-height: 1.0;
+    border-radius: 6px;
+    font-weight: 400;
+    transition-property: background-color,border-color,color,fill,stroke,opacity,box-shadow,transform;
+    transition-duration: var(--navbar-transition-duration);
+    min-width: 1.5rem;
+    transition-timing-function: ease-out;
+    height: auto;
+    font-size: 14px;
+    padding-inline-start: 2px;
+    padding-inline-end: 2px;
+    padding: 5px;
+    background: #E07807;
+    color: #ffffff;
+    position: absolute;
+    right: 20px;
+    top: 25px;
+    z-index: 1;
+    transform: translateX(50%);
+    border-color: transparent;
+    justify-content: flex-start;
+}
+.navbar-toggler .navbar-toggler-arrow {
+    flex: 0 0 auto;
+    text-align: center;
+    transition-duration: var(--navbar-transition-duration);
+    transition-property: color, transform;
+    transition-timing-function: ease-out;
+    transform: rotate(0);
+    width: 16px;
+    height: 16px;
+    fill: none;
+    border-color: transparent;
+}
+
+.navbar-header {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 240px; /* or whatever your full navbar width is */
+    z-index: 1031; /* Above the navbar-fixed-left */
+}
+
+.navbar-fixed-left, .main-content, .navbar-toggler .navbar-toggler-arrow {
+    transition-duration: var(--navbar-transition-duration);
+}
+
+.navbar-fixed-left, 
+#main-content, 
+.navbar-toggler .navbar-toggler-arrow, 
+.breadcrumb, 
+#footer {
+    transition-duration: var(--navbar-transition-duration);
+    transition-timing-function: ease-in-out;
+    transition-property: margin-left, width;
+}
+.banner-alert-area {
+    transition-duration: 0ms !important; 
+}
+.banner-alert-area.collapsed {
+    margin-left: 240px;
+}
+.breadcrumb.collapsed {
+    margin-left: 240px;
+}
```

Comment:
Are these correct? It feels weird to me that we'd add a `.collapsed` class to either of these areas.

Reply:
@glennmatthews - these are required so that the banner and breadcrumbs do not overlap the navbar-header and navbar-toggler button.  they keep the two elements in the same position even when the navbar is toggled to the collapsed state.